programs/clearing_house/src/math/orders.rs

```diff
@@ -281,6 +282,92 @@ pub fn order_satisfies_trigger_condition(order: &Order, oracle_price: u128) -> b
     }
 }
 
+pub fn is_order_risk_decreasing(
```

Comment:
can we use this in liquidate, where we close risk increasing orders? 👀 

so the result after closing orders is the worst case base asset amount is the current position?